lib/psci/psci_suspend.c

```diff
@@ -85,6 +85,14 @@ static void psci_suspend_to_pwrdown_start(unsigned int end_pwrlvl,
 	 */
 	cm_init_my_context(ep);
 
+	/*
+	 * Plat. management: Allow the platform to perform any early
+	 * actions required to power down the CPU, before we disable
+	 * the data cache.
```

Comment:
We only disable the data cache in the `HW_ASSISTED_COHERENCY = 0` case. I suggest the comma becomes a full stop and a new sentence added: "This might be useful for `HW_ASSISTED_COHERENCY = 0` platforms that can safely perform these actions with data caches enabled."